.gitignore

```diff
@@ -0,0 +1,2 @@
+*.save
+*~
```

Comment:
You are now ignoring backup files (~ files), but in your previous commit you introduced a bunch of them. You need to remove them from the commit.